markdown/org/docs/designs/opal/needs/en.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Opal Overalls: What You Need"
+---
+
+To make Opal, you will need the following:
+
+- Basic sewing supplies
+- Between 1 - 3 meters (1.1 - 3.3 yards) of a suitable fabric, depending on size and style ([see Fabric options](/docs/patterns/opal/fabric)).
+	- Long legs and a looser fit will increase fabric requirements. Pockets usually can be fit on the otherwise wasted portion of the fabric, and don't change the fabric length requirement.
+	- A pair of shortalls for a typical adult will need about 2 meters of fabric.
```

Comment:
```suggestion

<Warning>

	- Long legs and a looser fit will increase fabric requirements. Pockets usually can be fit on the otherwise wasted portion of the fabric, and don't change the fabric length requirement.
	- A pair of shortalls for a typical adult will need about 2 meters of fabric.
	
</Warning
```